src/modules/templates.js

```diff
@@ -0,0 +1,7 @@
+export const name = 'angular-templates';
```

Comment:
@Urigo `angular-meteor` creates empty module only if `angular-templates` is not available.